src/components/legend/label.tsx

```diff
@@ -22,19 +22,26 @@ import React, { MouseEventHandler } from 'react';
 
 interface LabelProps {
   label: string;
+  isSeriesHidden?: boolean;
   onClick?: MouseEventHandler;
 }
 /**
  * Label component used to display text in legend item
  * @internal
  */
-export function Label({ label, onClick }: LabelProps) {
+export function Label({ label, onClick, isSeriesHidden }: LabelProps) {
   const labelClassNames = classNames('echLegendItem__label', {
     'echLegendItem__label--clickable': Boolean(onClick),
   });
   return (
-    <div className={labelClassNames} title={label} onClick={onClick}>
+    <button
+      type="button"
+      className={labelClassNames}
+      title={label}
+      onClick={onClick}
+      aria-label={isSeriesHidden ? `Activate to show series in graph` : `Activate to hide series in graph`}
```

Comment:
`aria-label` replaces the contents of the button so we need to restate the `label` in here as well.

```suggestion
      aria-label={isSeriesHidden ? `${label}; Activate to show series in graph` : `${label}; Activate to hide series in graph`}
```

Reply:
Changed in dc373e8! Thanks!